W-14659026-SFDC-Bridge-se (#359)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/automation-tutorial-deploy.adoc

= Step 7: Publish the Automation to Exchange

> After finishing the testing phase, the process moves to the production phase. It is time to create an invocable run configuration, which publishes the process to Anypoint Exchange and enables MuleSoft Composer to trigger the execution of this process. 

Reviewer: ```suggestion
After finishing the testing phase, the process moves to the production phase. It is now time to create an invocable run configuration, which publishes the process to Anypoint Exchange and enables MuleSoft Composer to trigger its execution. 
```

> Before you can create an invocable run configuration, ensure that you configured your Anypoint Platform settings in RPA Manager. See xref:rpa-manager::organizationmanagement-settings.adoc#configure-anypoint-credentials[Configure Anypoint Platform Credentials] for instructions.

Reviewer: Global: For see and see also references, it is better to put what the user is going to see before the reference. This way, the user doesn't have to read the rest of the sentence if he/she doesn't need the info.

For instructions, see...

Author: Created a backlog story for this and similar edits: [General edits for better usability and consistency](https://gus.lightning.force.com/a07EE00001peMotYAE)
